apps/backend/.env.example

```
@@ -19,6 +19,8 @@ OIDC_ISSUER_URL=<your-oidc-provider-issuer-url>
 OIDC_CLIENT_ID=<your-oidc-provider-client-id>
 OIDC_CLIENT_SECRET=<your-oidc-provider-client-secret>
 OIDC_ALLOW_HTTP=<default=false|true> # Allow insecure HTTP connections to the OIDC provider. Only enable for local development if necessary.
+OIDC_SCOPE=<default=openid profile email> # Space-separated OIDC scopes to request; must include "openid".
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)